src/StockportWebapp/wwwroot/assets/sass/stockroom/modules/_footer.scss

```diff
@@ -133,11 +133,14 @@
 		margin-top: 0;
 
 		li {
-			padding-bottom: 0;
+            @media #{$tablet-only, $desktop} {
+				padding-bottom: 0;
+			}
 
             @media #{$mobile} {
                 &:last-of-type {
                     margin-bottom: 1em;
+					padding-bottom: 0;
```

Comment:
property not inline, the closing brackets at the end of the file are a bit wonky as well